src/SimpleWebRequest.ts

```diff
@@ -366,7 +366,23 @@ export abstract class SimpleWebRequestBase<TOptions extends WebRequestOptions =
         }
 
         const acceptType = this._options.acceptType || 'json';
-        this._xhr.responseType = SimpleWebRequestBase._getResponseType(acceptType);
+        const responseType = SimpleWebRequestBase._getResponseType(acceptType);
+
+        try {
+            this._xhr.responseType = responseType;
```

Comment:
Can you wrap this in _.attempt? try/catch causes the function to get de-optimized in JS engines

Reply:
Fixed, thank you.